dask_expr/_expr.py

```diff
@@ -2800,8 +2822,21 @@ def _simplify_down(self):
                 enforce_metadata=self.enforce_metadata,
                 kwargs=self.kwargs,
             )
-        else:
-            raise NotImplementedError()
+
+
+class ShiftAxis1(ShiftAxis0):
```

Comment:
Can you call map_partitions on the collection level?

We might not need a new class for now

Reply:
something like this? https://github.com/dask-contrib/dask-expr/pull/473/commits/27e37157e714d2e8af2e74e82aeb017398802e76